src/NuGet.Clients/VsExtension/RestoreManagerPackage.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+using System.Threading;
+using Microsoft.VisualStudio;
+using Microsoft.VisualStudio.ComponentModelHost;
+using Microsoft.VisualStudio.Shell;
+using NuGet.PackageManagement.VisualStudio;
+using NuGet.SolutionRestoreManager;
+using SystemTasks = System.Threading.Tasks;
+
+namespace NuGetVSExtension
```

Comment:
Perhaps this namespace could be revised to NuGet.Tools, as could the directory this project lives in.